website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
With the new `has_anonymous_link`, we lose the ability to override `auth.private_key` with the key that is present in the `view_only` request parameter. Do we depend on that functionality anywhere?